package.json

```diff
@@ -0,0 +1,40 @@
+{
+    "name": "gpii-grunt-mdjson-lint",
+    "description": "A plugin to lint JSON and JSON5 blocks within markdown documents.",
+    "version": "1.0.0",
+    "homepage": "http://github.com/GPII/grunt-mdjson-lint",
+    "author": "Tony Atkins <tony@raisingthefloor.org>",
+    "repository": {
+        "type": "git",
+        "url": "git://github.com/GPII/grunt-mdjson-lint.git"
+    },
+    "engines": {
+        "node": ">= 6.0.0"
+    },
+    "scripts": {
+        "test": "node node_modules/istanbul/lib/cli.js cover tests/all-tests.js"
+    },
+    "dependencies": {
+        "gpii-grunt-lint-all": "1.0.1-dev.20180706T153657Z.4cbbd61",
+        "infusion": "3.0.0-dev.20171006T195039Z.a128358",
+        "json5": "1.0.1",
+        "@textlint/markdown-to-ast": "6.0.3"
+    },
+    "devDependencies": {
+        "grunt": "~1.0.1",
```

Comment:
Please fix this version to the latest

Reply:
Done, I had wrong-headedly thought that was indicating compatibility for consumers and wanted to be flexible.